### DISCUSSION TITLE
UnicodeDecodeError, how to solution ?

### DISCUSSION THREAD

**Discussion by pedrober31** [pyinstaller/pyinstaller#6794] [Help] [answered]
I used the command **'pyinstaller --windowed GeoTech.py'** in my interface code, but I got this error: **UnicodeDecodeError: 'utf-8' codec can't decode byte 0xb0 in position 8200: invalid start byte.** 
I had already used the same command other times and never had this problem. 
I looked for some solutions in forums, but without success or I implemented what they asked in the wrong way, I appreciate any help.

**Comment by rokm** [comment, marked as answer]
You have a source file that is not saved in UTF-8 encoding but in your local encoding, and that file is using a non-latin character (for example, in ANSI encoding, 0xB0 is a degree (°) sign). 

So either:
* save the file as UTF-8 instead of your local encoding
* declare the file encoding as per [PEP263](https://peps.python.org/pep-0263), for example, adding `# coding=ansi` at the top of the file (replacing `ansi` with whatever encoding you're actually using)
* remove the non-latin characters from your code
